src/order/category/ωCPO.lean

```diff
@@ -0,0 +1,253 @@
+/-
+Copyright (c) 2020 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Simon Hudon
+-/
+
+import order.omega_complete_partial_order
+import order.category.Preorder
+
+/-! # Category of types with a omega complete partial order -/
```

Comment:
I added some documentation to the header of `omega_complete_partial_order`. This file is really just about bundling it as a category.

Reply:
I think it could easily contain a bit more content. After all, a lot more happens in between, where you show various functions are continuous, etc. That should be summarised in the module-doc, so people know what to expect in the file.

Reply:
Yes, I agree with Scott, there needs to be an explanation in this module doc.